flowtools/flowclr_sample_name/sample_simp.R

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/Rscript
+# Module for Galaxy
+# Adds sample information to a flowclustered file and binds 
+#
+#                  
+# Version 1
+# Emily Combe
+#
+#
+
+
+
+#-------------#
+# sample name #
+#-------------#
+
+
+library(flowCore)    
+
+sample_name <- function(input_file, sample_names){
+	flowtext <- list()
+	for(i in input_file){
```

Comment:
If you want input_file to behave as a list, but from Galaxy you are giving it a dataset collection, you need to (a) decompose the dataset collection as a concatenation of paths (possibly with commas) and then (b) transform that string of concatenated paths into a list in R.

Reply:
This will resolve the reading problem that you currently have, as currently the function is expecting to receive a list, but you are actually passing some undetermined string.